src/hooks/useBlocksInfo.ts

```diff
@@ -14,69 +14,34 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
 
-import { VoidFn } from '@polkadot/api/types';
-import { ApiPromise } from '@polkadot/api';
-import { useEffect } from 'react';
+import { SubscriptionInput } from '../types/subscriptionsTypes';
 import { useMountedState } from '../hooks/useMountedState';
-import logger from '../util/logger';
+import { useMakesSubscription } from '../hooks/useMakeSubscription';
```

Comment:
`useMakesSubscription` reads weird, maybe let's just call it
```suggestion
import { useSubscription } from '../hooks/useSubscription';
```

or `useApiSubscription`?